newIDE/app/src/MainFrame/Preferences/PreferencesContext.js

```diff
@@ -383,7 +379,6 @@ export const initialPreferences = {
     editorStateByProject: {},
     fetchPlayerTokenForPreviewAutomatically: true,
     previewCrashReportUploadLevel: 'exclude-javascript-code-events',
-    gamesListOrderBy: 'createdAt',
```

Comment:
removed the fact that we were saving the orderBy in the preferences, I feel it makes more sense to show always the latest one modified at the top as this will probably be the one you'll work on.

Reply:
Are we really sure it's worth losing the choice of the user everytime they navigate?

Reply:
I think that the user choice should be kept during a whole session but can be reset to the latest sort on each new session on GDevelop

Reply:
The reason I removed it is because when filtering/searching, the current game can disappear, because pushed down or filtered.
I was afraid people would not understand why they don't see the game they're working on at the top when arriving on the dashboard.
We can either, always show it at the top, or assume it's fine